sample-functions/NodeHelloEnv/handler.js

```diff
@@ -0,0 +1,13 @@
+"use strict"
```

Comment:
What is this sample for?

Reply:
I built this one to have a separate sample function to demonstrate accessing the ENV versus Secrets

Reply:
Just trying to work out if we still have the origin "ApiKeyProtected" example which used env-vars?

Reply:
I originally created this example for ENV and then update `ApiKeyProtected` to use secrets. I was asked to put the original `ApiKeyProtected` back and create a separate sample for secrets.